content/en/docs/how-tos/cluster-claim.md

```diff
@@ -183,6 +183,30 @@ stringData:
 type: Opaque
 ```
 
+### Set up RBACs for debugging
+
+The `ClusterRole` `hive-cluster-pool-admin` will [contains the permissions](https://github.com/openshift/hive/blob/master/docs/clusterpools.md#managing-admins-for-cluster-pools) to all resources created for a pool.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: hive-cluster-pool-admin-0
+  namespace: dptp-demo-cluster-pools
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: hive-cluster-pool-admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: petr
+```
+
+The pool owners need the permission of listing the namespace to filter out which namespaces are created for their pools.
+Login to [the `hive` cluster](https://console-openshift-console.apps.hive.9xw5.p1.openshiftapps.com/dashboards) and add the user into
+[the cluster-namespace-viewer group](https://github.com/openshift/release/blob/master/clusters/hive/admin_cluster-namespace-view_rbac.yaml).
```

Comment:
This is confusing to me. When we instruct users to create one `RoleBinding` already, could we make the process entirely straighforward by just instrcut them to create a 1) Group 2) RB that group to `hive-cluster-pool-admin` 3) CRB that group to `cluster-namespace-view` here, in a single step?

I also don't understand why we instruct the users to "login to the cluster and add the user into the group": the login seems to be a dead end, adding the user is GitOps.

Reply:
I will create another PR to follow the instructions for hypershift.

Removed the login instruction.
I was trying to tell how to determine the username by login.

Reply:
https://github.com/openshift/release/pull/19855